mantidproject/mantid
# Muon Manual Background Corrections

**Description of work.**
This PR adds the ability to do Manual background corrections in the Muon Analysis and FDA interfaces.

This feature is a requirement of the WIMDA Replacement project.

**To test:**
Open Muon Analysis
Load PSI 1249
Go to Corrections tab and select `Manual` background corrections
The background column should be enabled, and the status column should say 'No background corrections'
Change the background to a non-zero value. The correction should be applied and the status label updated to 'Correction success'
Try only changing the background for a single domain
Try the display options for displaying the data
Make sure auto corrections still work as expected

Fixes #31815

*No release notes are required because **I will add them in a separate PR along with other release notes for the new model fitting feature**.*

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
